vms/platformvm/vm_regression_test.go

```diff
@@ -686,19 +687,19 @@ func TestRejectedStateRegressionInvalidValidatorReward(t *testing.T) {
 	vm.state.SetCurrentSupply(constants.PrimaryNetworkID, defaultRewardConfig.SupplyCap/2)
 
 	newValidatorStartTime0 := vm.clock.Time().Add(executor.SyncBound).Add(1 * time.Second)
-	newValidatorEndTime0 := newValidatorStartTime0.Add(defaultMaxStakingDuration)
+	newValidatorEndTime0 := newValidatorStartTime0.Add(ts.MaxStakingDuration)
 
 	nodeID0 := ids.GenerateTestNodeID()
 
 	// Create the tx to add the first new validator
 	addValidatorTx0, err := vm.txBuilder.NewAddValidatorTx(
-		vm.MaxValidatorStake,
+		vm.MinValidatorStake,
```

Comment:
`ts.Keys[0]` is used to fund a subnet creation, whose fee is now non zero (it was due to wrong fork time setup). So `ts.Keys[0]` doesn't have enough balance left to fund a max stake validator, but it has to fund a min stake validator.
Hence the change. The alternative would be to use different keys. Not sure what we prefer